src/wallet/Wallet.ts

```diff
@@ -28,7 +35,12 @@ class Wallet {
    * Current network.
    */
   // @ts-ignore
-  network: Network = DEFAULT_NETWORK;
+  network: Network = DEFAULT_NETWORK.name as Network;
+
+  /**
+   * Current API endpoint for selected network
+   */
+  apiEndpoint: string = ((DEFAULT_NETWORK as unknown) as SelectedNetwork).apiBaseUrl;
```

Comment:
Does `DEFAULT_NETWORK.apiBaseUrl` give you a problem here? ahh typescript :)

Reply:
It does, because `DEFAULT_NETWORK` is defined in a JSON file that we import, so the type inference doesn't seem to work